packages/gatsby-theme/src/Apollo.ts

```diff
@@ -4,7 +4,7 @@ import { HttpLink } from 'apollo-link-http';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 import fetch from 'isomorphic-fetch';
 import config from '../config';
-import { getAuthToken } from './utils/auth';
+import { getAuthToken } from './utils/auth/auth';
```

Comment:
```suggestion
import { getAuthToken } from './utils/auth';
```

In cases like this, where you need to replace the `auth.ts` file with an auth folder, you could move the file into the auth folder and rename it to `index.ts`. This way, you won't have to change how you import the file-- when you specify a folder for importing, it will look for an index.ts file in that folder.